src/styles.ts

```diff
@@ -30,6 +32,7 @@ export const ComponentColors = {
     BACKGROUND_COLOR: Colors.LIGHT_GRAY,
     DISABLED_BUTTON_COLOR: Colors.LIGHT_GRAY,
     BUTTON_COLOR: Colors.BRAND_PURPLE,
+    SECONDARY_RECT_BUTTON_COLOR: Colors.LIGHTER_GRAY_2,
```

Comment:
What does RECT mean here?

Reply:
Rectangular

Reply:
Can it be simply `SECONDARY_BUTTON_COLOR`? With this I get the impression that there should be a `PRIMARY_RECT_BUTTON_COLOR` as well (and in that case I wouldn't shorten rectangular).

Reply:
that would be misleading, because, there is a `BUTTON_COLOR`, and it is not the corresponding secondary option color. I added a primary one and remove shortening.